test/utils.js

```diff
@@ -48,18 +46,18 @@ export function expectSuccessfulTransfer({
     balances[stepFromIndex] -= parseFloat(transaction.value);
     balances[stepToIndex] += parseFloat(transaction.value);
 
-    const { capacity } = edges.find((edge) => {
-      return (
+    edges.forEach((edge) => {
```

Comment:
isn't it the find() method ok?

Reply:
I tried `find` but for whatever reason it returns always `undefined` but using `foreach` it actually finds a match... I would investigate more but it works with foreach and it's just for these test so I thought it's ok to have not with super performance